Example/Example Swift/ComponentAPICoordinator.swift

```diff
@@ -171,8 +171,11 @@ extension ComponentAPICoordinator {
     @objc fileprivate func showAnalysisScreen() {
         guard let document = documentRequests.first?.document else { return }
         
+        if analysisScreen == nil {
```

Comment:
Is there a valid case when the analysis screen might not be nil here?
Later on in the method you push it to the navigation stack. If the analysis screen is already existing, it is also quite likely that it's somewhere in the view hierarchy. If you try to push it (again), it will fail/crash/burn.

Reply:
When showing an error, I just answered you about it below.